docker-compose.yml

```diff
@@ -28,15 +28,43 @@ services:
       - ./api/test:/opt/landgriffon-api/test
     env_file: .env
     environment:
+      - NODE_OPTIONS=${HEAP_SIZE}
```

Comment:
???

Reply:
Ah yes, ideas of a mad man when feeling alone in the darkness. I totally forgot this, thanks